custom_components/recycle_app/__init__.py

```
@@ -59,16 +59,30 @@
     async def async_update_collections() -> dict[str, list[date]]:
         """Fetch data."""
         _LOGGER.debug("Update collections")
-        return await hass.async_add_executor_job(
-            api.get_collections, zip_code_id, street_id, house_number
-        )
+        retry = coordinator.update_interval
+        try:
+            coordinator.update_interval = None
+            return await hass.async_add_executor_job(
+                api.get_collections, zip_code_id, street_id, house_number
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)